tools/fs/files.ts

```diff
@@ -1741,10 +1743,14 @@ wrapFsFunc<[string], string>("realpath", fs.realpathSync, [0], {
 });
 
 export const readdir =
-wrapFsFunc<[string], string[]>("readdir", fs.readdirSync, [0], {
+wrapFsFunc<[string, { encoding: 'utf-8' | null, withFileTypes: true }], string[] | fs.Dirent[]>("readdir", fs.readdirSync, [0], {
```

Comment:
I am struggling with finding the correct type for this. This is the reason the tests are failing on Circle CI.